picard/file_dilaog.py

```diff
@@ -0,0 +1,274 @@
+from PyQt5.QtGui import *
+from PyQt5.QtWidgets import *
+from PyQt5.QtCore import *
+
+import os,sys
+if os.name == 'nt':
+    import win32api, win32con
+from datetime import datetime
+
+def Is_hidden(file_temp):
+    if os.name== 'nt':
+        attribute = win32api.GetFileAttributes(file_temp)
+        return attribute & (win32con.FILE_ATTRIBUTE_HIDDEN | win32con.FILE_ATTRIBUTE_SYSTEM)
+    else:
+        return file_temp.startswith('.')
+
+class ImgWidget1(QLabel):
+    """
+    Class used to show picture on Table Row
+    """
+    def __init__(self, imagePath,parent=None):
+        super(ImgWidget1, self).__init__(parent)
+        pic = QPixmap(imagePath)
+        self.setPixmap(pic)
+
+class File_dialog(QDialog):
+    """
+    Main class used for the dialog
+    """
+    def __init__(self,wi,he,path,parent=None):
+        self.path = path
+        self.formats=["MP3","FLAC","OGG","M4A","WMA","WAV","MP4","AAC","flv"]
+        self.formats_lower=[]
+        for i in self.formats:self.formats_lower.append(i.lower())
+        self.formats_show=["MP3","FLAC","OGG","M4A","WMA","WAV","MP4","AAC","flv"]
+        super(File_dialog,self).__init__(parent)
+        self.initUI(wi,he)
+
+    def initUI(self,wi,he):
+        self.moveBack = QPushButton(self)
+        self.moveBack.resize(35,30)
+        self.moveBack.setIcon(QIcon('../resources/img-src/moveBack24_24.png'))
+        self.moveBack.setIconSize(QSize(35,30))
+        self.moveBack.move(.17*wi,10)
+        self.moveBack.clicked.connect(self.moveBack_action)
+        self.moveNext = QPushButton(self)
+        self.moveNext.resize(35,30)
+        self.moveNext.setIcon(QIcon('../resources/img-src/moveNext24_24.png'))
+        self.moveNext.setIconSize(QSize(35,30))
+        self.moveNext.move(.17*wi+35,10)
+        self.moveNext.clicked.connect(self.moveNext_action)
+
+        self.selectBtn = QPushButton("Select It",self)
+        self.selectBtn.resize(100,30)
+        self.selectBtn.move(wi-0.09*wi,10)
+        self.selectBtn.clicked.connect(lambda:self.select_action(True))
+
+        self.selectBtn1 = QPushButton("Open Folder",self)
+        self.selectBtn1.resize(150,30)
+        self.selectBtn1.move(wi-.219*wi,10)
+        self.selectBtn1.clicked.connect(lambda:self.open_action(True))
+
+        self.lineedit = QLineEdit("Search here",self)
+        self.lineedit.resize(250,30)
+        self.lineedit.move(wi-.51*wi,10)
```

Comment:
Please do not set such arbitrary numbers, instead make a .ui file via Qt designer and convert it to _ui file via `python setup.py build_ui` 
I would prefer if you separate the UI and the logic parts like the rest of the UI components do.